consensus/istanbul/backend/engine_test.go

```diff
@@ -270,7 +270,7 @@ func makeHeader(parent *types.Block, config *istanbul.Config) *types.Header {
 	header := &types.Header{
 		ParentHash: parent.Hash(),
 		Number:     parent.Number().Add(parent.Number(), common.Big1),
-		GasLimit:   core.CalcGasLimit(parent, parent.GasLimit(), parent.GasLimit()),
+		GasLimit:   core.CalcGasLimit(parent, nil),
```

Comment:
I think we should remove this field from the block header

Reply:
Perhaps this should be in another PR, changing the header seems to break many things in the tests.

Reply:
Works for me!